holder.yml

```
@@ -268,7 +268,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/ProvePresentationResponse"
+                  $ref: "#/components/schemas/ProvePresentationResponse"
```

This seems like an accidental change.

```suggestion
                $ref: "#/components/schemas/ProvePresentationResponse"
```